pilot/pkg/xds/ads.go

```diff
@@ -80,20 +81,11 @@ type Connection struct {
 	// Currently based on the node name and a counter.
 	conID string
 
-	// proxy is the client to which this connection is established.
-	proxy *model.Proxy
-
 	// Sending on this channel results in a push.
-	pushChannel chan *Event
+	pushChannel chan any
```

Comment:
Is there other type other than Event

Reply:
In pilot, no. In node agent, yes, it's a different type.

Reply:
IC